_includes/community-help-band.html

```diff
@@ -0,0 +1,68 @@
+<div class="component-wrapper contributions-band">
```

Comment:
In the /community page we lost:

- Quarkus QTips reference
- publications

Also I am not liking the translate in your language: that's a lot of work and infra from us that we don't have the bandwidth for.

Reply:
This is a content thing, so it's totally up to you guys if you want to remove it or not. @adelasofia is removing Translate.

We'll add in custom icons for QTips and Publications but they might come in a separate PR.

Reply:
@emmanuelbernard, are these icons and descriptions OK?
![Screen Shot 2019-11-04 at 3 22 21 PM](https://user-images.githubusercontent.com/10758508/68159243-02558300-ff17-11e9-9341-a8d657d287ff.png)

Reply:
@jkbrock  The Quarkus _community_ publishes thoughtful...
The rest looks good.

Reply:
I fixed the text

Reply:
I was on it but you already fixed it, thanks!